pdu/PDU.go

```diff
@@ -21,6 +21,8 @@ type PDU interface {
 	// GetResponse PDU.
 	GetResponse() PDU
 
+	GetSettedResponse() *PDU
```

Comment:
How about renaming to `GetCustomResponse()`

Reply:
Agree done in 46a3cb246616208c70588a33dd0b2f560b355f49